corona-warn-app/cwa-app-android
# Low Risk after update from 1.7.0 RC8 (in increased risk state) to 1.8.0 RC2 (EXPOSUREAPP-3988)

Only run risk level calculation when we actually got data from enf v2.

**Test**
- expose 1.7.0RC8 devices to each other
- submit positive test result (risk level card turns red)
- update apps to 1.8.0

expected:
- risk level card remains unchanged until it is time to download new diagnosis keys
- after download of diagnosis keys & risk level calculation:
  - number of exposure windows > 0
  - risk level card is still red
